src/components/ButtonBox/index.jsx

```diff
@@ -0,0 +1,43 @@
+import React from "react";
+import Button from "../Button/index.jsx";
+import styles from "./styles/styles.module.scss";
+
+const ButtonBox = ({
+  PreviousButtonType,
+  PreviousButtonText,
+  previousPath,
+  nextButtonType,
+  nextButtonText,
+  nextPath,
+}) => {
+  let buttonBox = (
+    <div className={styles.one_button_box}>
+      <Button
+        text={nextButtonText}
+        type={nextButtonType}
+        path={nextPath}
+      ></Button>
+    </div>
+  );
+
+  if (previousPath !== undefined) {
+    buttonBox = (
+      <div className={styles.two_button_box}>
+        <Button
+          text={PreviousButtonText}
+          type={PreviousButtonType}
+          path={previousPath}
+        ></Button>
```

Comment:
```suggestion
        <Button
          type={PreviousButtonType}
          path={previousPath}
        >{PreviousButtonText}</Button>
```
And then use `children` in the button component.

Reply:
doneeeeee